src/leveldbwrapper.h

```diff
@@ -16,6 +16,27 @@
 #include <leveldb/db.h>
 #include <leveldb/write_batch.h>
 
+// compression is purposely set to leveldb::kNoCompression because stored data is
+// not compressible, being mainly criptographic data like hashes, keys, signatures
+// moreover, compression library (Snappy) used by LevelDB is made not available for
+// zend (identifier SNAPPY being always undefined); hence, even enabling compression
+// at this point would be useless (unless Snappy is downloaded, compiled and linked
+// as external dependency; that's not the case)
+constexpr leveldb::CompressionType DB_COMPRESSION = leveldb::kNoCompression;
+
+// https://github.com/bitcoin/bitcoin/pull/12495
+// On most platforms the default setting of max_open_files (which is 1000)
+// is optimal. On Windows using a large file count is OK because the handles
+// do not interfere with select() loops. On 64-bit Unix hosts this value is
+// also OK, because up to that amount LevelDB will use an mmap
+// implementation that does not use extra file descriptors (the fds are
+// closed after being mmaped).
+// Increasing the value beyond the default is dangerous because LevelDB will
+// fall back to a non-mmap implementation when the file count is too large.
+// On 32-bit Unix host we should decrease the value because the handles use
+// up real fds, and we want to avoid fd exhaustion issues.
+constexpr unsigned int DB_MAX_OPEN_FILES = 1000;
+
```

Comment:
I agree with your opinion; I've moved the comments inside `GetOptions`, I prefer making `options.max_open_files = 1000` explicit rather than relying on the default of `leveldb::Options` structure.